components/organisms/CriteriaGrid.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
+
+import { useTranslation } from "next-i18next";
+
+export function CriteriaGrid(props) {
+  const { t } = useTranslation("common");
+  return (
+    <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-10 px-2  md:px-0">
+      {props.children}
+    </div>
```

Comment:
I know this doesn't look like much here, but the grid really is better than the other one!